src/ray/common/ray_config_def.h

```diff
@@ -394,6 +394,10 @@ RAY_CONFIG(int64_t, max_placement_group_load_report_size, 1000)
 /// external storage.
 RAY_CONFIG(std::string, object_spilling_config, "")
 
+/// Log an ERROR-level message about spilling every time this amount of bytes has been
+/// spilled, with exponential increase in interval. This can be set to zero to disable.
+RAY_CONFIG(int64_t, verbose_spill_logs, 2L * 1024 * 1024 * 1024)
```

Comment:
I feel this should be off by default since spilling as error message (this is a normal thing) is bad ux for normal ray users. People can manually enable them during debugging.

Reply:
I really disagree here, it's more important to surface what Ray is doing. There are two scenarios:
- spilling is expected: minor nuisance that can be disabled
- spilling is unexpected: major benefit to the user

Reply:
Yea, I'm totally fine with surfacing what Ray is doing. It's just the log level that concerns me a bit: not sure if we have clearly defined meaning for each log level (something like https://stackify.com/logging-levels-101/), but I just think spilling is not an error and print it out as error may make operators panic (especially if it's running in production).

Reply:
I can try routing this to a driver message instead, I think that should be doable.